bff/test/Duende.Bff.Tests/TestHosts/BffHost.cs

```diff
@@ -156,7 +156,7 @@ private void Configure(IApplicationBuilder app)
                         context.Request.Path.Value ?? "/",
                         context.User.FindFirst("sub")?.Value,
                         context.User.FindFirst("client_id")?.Value,
-                        context.User.Claims.Select(x => new ClaimRecord(x.Type, x.Value)).ToArray())
+                        context.User.Claims.Select(x => new TestFramework.ClaimRecord(x.Type, x.Value)).ToArray())
```

Comment:
Does there need to be a separate ClaimRecord in TestFramework? Can it get in on the shared goodness?

Reply:
Actually it can and it should and it is now :)